lib/rubygems/gem_runner.rb

```diff
@@ -81,6 +89,14 @@ def do_configuration(args)
     Gem::Command.extra_args = Gem.configuration[:gem]
   end
 
+  def warn_on_outdated
+    current_version = Gem.rubygems_version
+    rubygems_latest = Gem.latest_rubygems_version
+    if rubygems_latest > current_version
```

Comment:
What you think of removing the variables and just use `Gem.rubygems_version` and `Gem.latest_rubygems_version`?